### DISCUSSION TITLE
Support for wandb (weights and biases)

### DISCUSSION THREAD

**Discussion by jpcenteno80** [Project-MONAI/MONAI#3155] [Ideas]
Hi, I was wondering if there are plans to support [`wandb`](https://wandb.ai/site) in the future. I think it is a better tool than `MLFlow` for not just tracking metrics, but also for tracking artifacts and the code that produced those artifacts. You can track data used to train the model, the trained model, or any other large things that might not fit in Git. With the end goal of having reproducible results down the road. I've implemented my own `wandb` handler but now I'm stuck posting the validation metrics in sync with the global epoch being tracked by the trainer (since trainer and evaluator are a bit independent of each other). Do you have any tips on how to pass to a handler inside a `SupervisedEvaluator` the epoch just completed by the `SupervisedTrainer`, so that the `SupervisedEvaluator` metrics are posted at the correct training epoch? Thanks!

**Comment by adamaji** [comment]
Hi @jpcenteno80 , I recognize this is an old post, just wondering if you got any further with your `wandb` handler? 

Our group is also interested in using `wandb` through the MONAILabel training set up, and it seems like to get tracking working there we'd also need a handler. @wyli would there be interest in this addition to the repo?

cc @thomaskierski

**Reply by wyli** [reply]
Sure, contributions are welcomed, new handlers similar to https://github.com/Project-MONAI/MONAI/pull/6013 would be helpful. 

Also have you considered the handler from ignite? https://github.com/pytorch/ignite/blob/master/ignite/contrib/handlers/wandb_logger.py If that implementation can solve the problem, then there's no need to reimplement it in monai.

**Reply by Nic-Ma** [reply]
CC @binliunls to help reviewing and testing the PR later.

Thanks.
